### PR TITLE
Create Palindrome_Substrings.py

### LINKED ISSUE #73: Palindromic Substrings
**This issue is open for C Implementation**
Palindromic substrings

Make a program to find all possible continuous subsequences of an array and print the ones which are a palindrome.

## Bug Reports
Please replace this line with a brief summary of your issue AND if reporting a build issue include the link.

- [ ] Didn't run?
- [ ] Permission Issues?
- [ ] Broken Links
- [ ] Other Issues? (Add Here)

## Any links, resources or references to solve the issue?
- [ ] Yes

If yes, please the resources in the following format:
- Resource 1
- Resource 2

- [ ] No

### PR DESCRIPTION
**Pull request by deVedant**
# Palindrome Substrings

Fixes #73 

## Written a program in python to find all palindrome substrings in a string input by user. 

-   [x] New feature (non-breaking change which adds functionality)

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generates no new warnings

### CODE REVIEW

File: Python/Palindrome_Substrings.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+def find_all_palindrome_substrings(input):
+  count = 0
+  for i in range(0, len(input)):
+    count += find_palindromes_in_sub_string(input, i - 1, i + 1)
+    count += find_palindromes_in_sub_string(input, i, i + 1)
+
+  return count
+
+
+s = input("Enter a string : ")
```
Comment (kaustubhgupta) [inline Python/Palindrome_Substrings.py]: - use driver code for better code readability. Refer this page for the same: 
https://docs.python.org/3.7/library/__main__.html

- Also, make the entry of this program in the Python directory readme.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
@deVedant Code is fine now, can you update the Python directory readme too so that it can be merged?

**Comment by deVedant** [comment]
> @deVedant Code is fine now, can you update the Python directory readme too so that it can be merged?

Actually this is my another account and I'm working on my mobile. I am some 28 commits behind master 😅

Issue is pulling(fetching) changes, as this feature ain't on mobile.
Could you please do it for me?

**Comment by kaustubhgupta** [comment]
@deVedant Okay, we can make an exception for you. Code reviewed and accepted!

**Pull request merged**